gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/TestUtility.java

```diff
@@ -28,6 +32,32 @@ public static CV createCv(String code) {
         return createCv(code, "", "");
     }
 
+    public static CD createCd(String code, String codeSystem, String displayName) {
+        final CD cd = new CD();
+        cd.setCode(code);
+        cd.setCodeSystem(codeSystem);
+        cd.setDisplayName(displayName);
+        return cd;
+    }
+
+    /**
+     * An EHR Extract has a cardinality of one to many components, each component (based
+     * of the UK05 schema) can contain one and only one EHR Folder. This utility method provides
+     * a means of extracting ALL components from within a target EHR Folder.
+     * @param extract The EHR Extract.
+     * @param extractComponentIndex The index of the RCMRMT030101UKComponent which houses the EHR Folder.
+     * @return A list of RCMRMT030101UKComponent3.
+     */
+    public static List<RCMRMT030101UKComponent3> getEhrFolderComponent(RCMRMT030101UKEhrExtract extract,
```

Comment:
Let's pluralise this!

```suggestion
    public static List<RCMRMT030101UKComponent3> getEhrFolderComponents(RCMRMT030101UKEhrExtract extract,
```